python_modules/libraries/dagster-pyspark/dagster_pyspark_tests/test_types.py

```diff
@@ -51,11 +69,9 @@ def return_df(_):
                         'outputs': [
                             {
                                 'df': {
-                                    file_type: {
-                                        'path': temp_path,
-                                        'mode': 'overwrite',
-                                        'compression': 'gzip',
-                                    }
+                                    file_type: dict(
+                                        {'mode': 'overwrite', 'compression': 'gzip',}, **options,
```

Comment:
I'm noticing a syntax error here for in the python 2.7 tests: https://buildkite.com/dagster/dagster/builds/14227#9a8db1d9-518e-4491-87a8-d247ca90946c

Reply:
The comma at the end of the line caused this error.
 